src/sentry/integrations/bitbucket/integration.py

```diff
@@ -24,26 +27,43 @@
     'account',
     'issue:write',
     'repository',
+    'repository:admin',
     'team',
     'webhook',
 )
 
 
-class BitbucketIntegration(Integration):
+class BitbucketIntegration(Integration, BitbucketIssueSyncMixin):
     def get_client(self):
         return BitbucketApiClient(
             self.model.metadata['base_url'],
             self.model.metadata['shared_secret'],
             self.model.external_id,
         )
 
+    def get_repo(self, repo=None, project_id=None):
```

Comment:
Does this get used yet?

Reply:
I'm not sure what that was for. Good catch. 👍